Client/DeviceCheck/Cryptography.swift

```diff
@@ -269,6 +256,25 @@ public class Cryptography {
     return CryptographicKey(key: pKey, keyId: id)
   }
   
+  /// Deletes the key with the specified ID from the secure-enclave and keychain
+  @discardableResult
+  public class func delete(id: String) -> Error? {
```

Comment:
Does it need to be public?

Reply:
Yeah because if it needs to be used outside of BraveRewards, it should be public otherwise it can't access it.